### DISCUSSION TITLE
Health Check for "*Sink*" failed: deadline has elapsed

### DISCUSSION THREAD

**Discussion by krisss24** [vectordotdev/vector#20926] [Q&A]
I encountering an error when I running the "vector validate --config-yaml" command, the error is health check for "sink" failed: deadline has elapsed. The sink I I used is vector, has anyone have encounter this error? 

Also, I can telnet the port of the vector from the server where the vector agent installed.

error:
![image](https://github.com/user-attachments/assets/4d17c74e-6f75-4f80-9a11-d18f0703a51e)

Vector config:
![image](https://github.com/user-attachments/assets/5ce6e7b7-fce5-4b03-8038-46da953298ee)

**Comment by jszwedko** [comment]
I think that could happen if the `vector` sink was enable to connect to the configured endpoint.

**Reply by krisss24** [reply]
Do you mean the TLS connection should be enable?
